web/ui/src/common/authService.js

```diff
@@ -24,6 +24,10 @@
              */
             setLoggedIn: setLoggedIn,
 
+            auth0login: function () {
+                angularAuth0.authorize();
+            },
```

Comment:
I'm not sure if this automatically tries to reauth before trying to create a new auth0 session or not, but we need to do that